frontend/src/Term.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * Represents a single term of a group.
+ * Most groups have only a single term, but not all do.
+ */
+export default class Term {
+  constructor (group, teacher, day, timeFrom, timeTo, weekParity) {
+    /**
+     * Group this term belongs to
+     */
+    this.group = group
+
+    /**
+     * Name of the teacher
```

Comment:
nepotřebný komentář

Reply:
Probírat se zapomenutým kódem je děs a je to o to horší, když člověk neví, jaký co má typ. Přidám tam, že je to string.